spec/features/onboarding_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'Onboarding Page', type: :feature do
   describe 'when a user has logged in for the first time' do
     before(:each) do
-      user = { 'id' => '1', 'name' => 'test' }
+      user = { 'id' => '1', 'name' => 'test', 'username' => 'pitzelalex' }
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
 
```

Comment:
Similar to the test above, this stub prevents us from actually testing the feature. How it is written here, the user already has a username, you are not adding a username to current user with this onboarding test. I am assuming you are aware, and are addressing this in the issues mentioned, just leaving a note / making sure.